src/cmd/linuxkit/cache/push.go

```diff
@@ -7,65 +7,9 @@ import (
 	namepkg "github.com/google/go-containerregistry/pkg/name"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/linuxkit/linuxkit/src/cmd/linuxkit/registry"
+	log "github.com/sirupsen/logrus"
 )
 
```

Comment:
For the curious, we haven't used `PushWithManifest()` in some time; all of the functionality therein already is in just `Push()`.